docs/guides/import-data-with-api.mdx

```diff
@@ -254,3 +254,15 @@ curl --location --request GET \
 
 </TabItem>
 </Tabs>
+
+:::tip
```

Comment:
Instead of a tip, how about a new section _Adding Scenes to Connect_? This way it's listed in the TOC and allows for hyperlinking.

Reply:
I'm going to leave it as-is for now, because that might lead to a larger restructuring of how this information is grouped. I'm not sure it makes sense to have a full section called _Adding Scenes to Connect_, but not share how to create a scene until two pages later. I think it's a good idea for the future though!